docs/community/maintainers-guide/releasing.rst

```diff
@@ -0,0 +1,224 @@
+A new minor release ``1.3.0`` of this module will not affect
+``invenio 3.3``.
+
+If this is not the case, you will have to make sure that the new release
+of the module ``foo`` will not introduce backward incompatible changes,
+because it will be installed by ``invenio 3.3``, potentially breaking
+existing Invenio installations.
+
+Module release
+--------------
+
+How to release an Invenio module.
+
+**Python module**
+
+1. Make sure that all relevant PRs have been merged to ``master``.
+2. Make sure that the build on ``master`` branch is passing.
+3. Prepare release notes in
+    - Update the ``CHANGE.rst`` file, making sure that you describe changes,
+      highlighting backward incompatible ones, and that the release date
+      is correct. Pay attention how you write, keep consistency with previous
+      sentences forms.
+4. Bump version in ``version.py`` (follow semantic versioning).
+5. Create a release commit (message pattern:
+   ``release: v<major>.<minor>.<patch>``).
+6. Create tag and push release:
+    - ``git tag v<major>.<minor>.<patch>``
+    - ``git push inveniosoftware master v<major>.<minor>.<patch>``
+7. Regularly check ``Travis`` and ``PyPI`` to ensure the release went through.
+
+**JavaScript/React module**
+
+1. Make sure that all relevant PRs have been merged to ``master``.
+2. Make sure that the build on ``master`` branch is passing.
+3. Prepare release notes in
+    - Update the ``CHANGE.md`` file, making sure that you describe changes,
+      highlighting backward incompatible ones, and that the release date
+      is correct. Pay attention how you write, keep consistency with previous
+      sentences forms.
+4. Bump version in ``package.json`` (follow semantic versioning).
+5. Regenerate the ``package-lock.json``:
+    - ``rm package-lock.json``
+    - ``npm install``
+6. Create a release commit (message pattern:
+   ``release: v<major>.<minor>.<patch>``).
+7. Create tag and push release:
+    - ``git tag v<major>.<minor>.<patch>``
+    - ``git push inveniosoftware master v<major>.<minor>.<patch>``
+8. Regularly check ``Travis`` and ``NpmJS`` to ensure the release went through.
+
+**Backport changes to a maint release**
+
+When you need to create a new release for a maintained previous release,
+you will have to do the extra step of backporting the changes to the
+maintenance branch.
+
+After the procedure described above, you will have to:
+
+1. Checkout the ``maint`` branch and create a new release branch:
+    - ``git checkout maint-<major>.<minor>``
+    - ``git checkout -b rel-v<major>.<minor>.<patch>``
+2. Cherry-pick the commits that you need to backport, resolve any conflict.
+   (yes, cherry-pick :))
+3. Run tests.
+4. Createa new commit with updated ``changes`` file and bumped version,
+   as described above.
+5. Issue a pull request against the **maintenance branch** (
+   ``maint-<major>.<minor>``).
+6. If Travis fails:
+    - Fix issue and ensure head commit is the release commit (i.e. rebase if
```

Comment:
I see you highlight important info in bold right? I would put in bold the part "ensure head commit is the release commit"